vagrant-pxe-harvester/ansible/roles/harvester/templates/config-join.yaml.j2

```diff
@@ -6,6 +6,10 @@ token: {{ settings['harvester_config']['token'] }}
 os:
   hostname: harvester-node-{{ node_number }}
   ssh_authorized_keys:
+  dns_nameservers:
```

Comment:
The /etc/resolv.conf file has `nameserver -` at the last line.

```
harvester-node-0:~ # cat /etc/resolv.conf
### /etc/resolv.conf is a symlink to /var/run/netconfig/resolv.conf
### autogenerated by netconfig!
#
# Before you change this file manually, consider to define the
# static DNS configuration using the following variables in the
# /etc/sysconfig/network/config file:
#     NETCONFIG_DNS_STATIC_SEARCHLIST
#     NETCONFIG_DNS_STATIC_SERVERS
#     NETCONFIG_DNS_FORWARDER
# or disable DNS configuration updates via netconfig by setting:
#     NETCONFIG_DNS_POLICY=''
#
# See also the netconfig(8) manual page and other documentation.
#
### Call "netconfig update -f" to force adjusting of /etc/resolv.conf.
nameserver 192.168.0.254
nameserver 8.8.8.8
nameserver -
```

Here is the generated config file:
```
harvester-node-0:~ # cat /oem/harvester.config
<...>
  ntpservers:
  - 0.suse.pool.ntp.org
  - 1.suse.pool.ntp.org
  dnsnameservers:
  - 192.168.0.254
  - 8.8.8.8 - ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ==
    vagrant insecure public key
  wifi: []
<...>
```
We might need a new line after the name `dns_nameservers` field.

Reply:
You were right. There was a typo where I removed the label line. I pushed a commit to fix the issue.